src/Dispatcher.php

```diff
@@ -11,6 +11,7 @@
 use Penny\Exception\MethodNotAllowed;
 use Penny\Exception\RouteNotFound;
 use Psr\Http\Message\RequestInterface;
+use Penny\Route\RouteInfo;
 
 class Dispatcher
```

Comment:
Don't know... It just return a `RouteInfo` there is no dispatch in place, right?